lib/ui/theme/cobble_theme.dart

```diff
@@ -145,6 +145,24 @@ class CobbleTheme {
           ),
         ),
       ),
+      tabBarTheme: TabBarTheme(
+        labelStyle: textTheme.bodyText2!.copyWith(
+          fontSize: 16,
+        ),
+        labelColor: scheme.text,
+        unselectedLabelStyle: textTheme.bodyText2!.copyWith(
+          fontSize: 16,
+        ),
+        labelPadding: EdgeInsets.only(top: 5, bottom: 4),
+        unselectedLabelColor: scheme.muted,
+        // Indicator should be rounded and not straight, implement that later
```

Comment:
Consider adding `// TODO Indicator should...` if that's something that needs to be implemented.